test-integration/scripts/12-generate-project.sh

```diff
@@ -11,22 +11,38 @@ if [ "$JHI_FOLDER_APP" == "$HOME/app" ]; then
     cp "$JHI_INTEG"/configstore/*.json "$HOME"/.config/configstore/
 fi
 
-#-------------------------------------------------------------------------------
-# Generate UAA project with jhipster
-#-------------------------------------------------------------------------------
-if [[ "$JHI_APP" == *"uaa"* ]]; then
-    mkdir -p "$JHI_FOLDER_UAA"
-    cp -f "$JHI_SAMPLES"/uaa/.yo-rc.json "$JHI_FOLDER_UAA"/
-    cd "$JHI_FOLDER_UAA"
-    jhipster --force --no-insight --with-entities --skip-checks --skip-git --skip-commit-hook --from-cli
-    ls -al "$JHI_FOLDER_UAA"
-fi
+if [[ "$JHI_ENTITY" == "jdl" ]]; then
+    #-------------------------------------------------------------------------------
+    # Generate with JDL
+    #-------------------------------------------------------------------------------
+    mkdir -p "$JHI_FOLDER_APP"
+    cp -f "$JHI_SAMPLES"/"$JHI_APP"/*.jdl "$JHI_FOLDER_APP"/
+    cd "$JHI_FOLDER_APP"
+    # Note: --no-insight doesn't work here
+    # there is still the message: May JHipster anonymously report usage statistics to improve the tool over time? (Y/n)
+    # so the CI is stuck with --no-insight
+    jhipster import-jdl *.jdl --force-insight
```

Comment:
Yes, that's why it is still WIP with this comment, so I don't forget to remove it and switch to `--no-insight`.
In fact, the argument is correct in import-jdl but there is a bug, the sub gen jhipster:app ignores the initial flag.
At start: this.noInsight=true
Then, the value changed: this.noInsight=false

```
18:01:06 in ~/tmp/16-app-jdl took 2s 
➜ jhipster import-jdl *.jdl --no-insight --with-cli
Using JHipster version installed globally
===========>true
Executing import-jdl toto.jdl
Options: insight: false, withCli: true, from-cli: true, with-cli: true
Found .yo-rc.json on path. This is an existing app
The JDL is being parsed.
warn: An Entity name 'User' was used: 'User' is an entity created by default by JHipster. All relationships toward it will be kept but any attributes and relationships from it will be disregarded.
No change in entity configurations, no entities were updated.
The JDL has been successfully parsed
Generating 1 application.
Generating application HealthPoints in a new parallel process
Executing jhipster:app on /home/pgrimaud/tmp/16-app-jdl
Options: withCli: true, fromCli: true, from-cli: true, with-cli: true
===========>false
```

Reply:
`--no-insight` fixed in last commits